### ISSUE TITLE
Match against bookmarks' tags

### ISSUE THREAD

**Issue by alphapapa** [josh-berry/tab-stash#532]
### Problem Statement

Perhaps I'm "misusing" Tab Stash, but its UI is so good that I find myself wanting to use it more and more.  Something I'd like to be able to do is search through stashed tabs by tags (i.e. since they are bookmarks, internally, they can be tagged like any other Firefox bookmark).

### Preferred Solution(s)

For example, if a stashed tab's bookmark has the tag `jazz`, but its title is `Emmet Cohen`, it would be nice if that stashed tab appeared in the search results for the query `jazz`.

### Alternative Solution(s)

I could imagine that some users might prefer to make tag searches explicit, so maybe an option could be used to require a prefix, like `tag:jazz`.

### Additional Context

Thanks again for your work on this.  It just keeps getting better.  (And I continue to be thankful that I don't have to use Chrome(-based) browsers for personal browsing!)

### Vote for This Issue

- [X] _Readers: If you are also interested in seeing this feature be developed, please vote for it by giving the ORIGINAL POST a thumbs-up using the :smiley: button below.  You are welcome to leave comments and discuss the feature request, but "Me too!" comments are not counted by the voting system._

**Comment by josh-berry** [comment]
I agree this would be super useful (and there are other things I'd like to be able to use it for, too, like tracking which bookmarks are in which containers). Unfortunately, Firefox just doesn't expose the tags feature to extensions at all--there's no way for an extension to see or modify tags on bookmarks. Perhaps one day, if the API ever gets added!
